ci/ray_ci/ml.tests.yml

```diff
@@ -1,5 +1,6 @@
 flaky_tests:
   # non-gpu tests
   - //python/ray/train:horovod_cifar_pbt_example
-  # doc tests
-  # gpu tests
+  # rely on scikit-optimize that cannot be upgraded
+  - //python/ray/tune:test_sample
```

Comment:
ml team will deprecate these tests later on

Reply:
Let me put up a PR to remove `scikit-optimize` now.

Reply:
PR here: https://github.com/ray-project/ray/pull/43969

Reply:
love it!